src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/Common/DbConnectionOptions.cs

```diff
@@ -72,39 +57,33 @@ internal bool ConvertValueToIntegratedSecurityInternal(string stringValue)
             }
         }
 
-        public int ConvertValueToInt32(string keyName, int defaultValue)
+        public int ConvertValueToInt32(Dictionary<string, string> parsetable, string keyName, int defaultValue)
```

Comment:
Can be made `internal` too?

Reply:
I am using ReSharper and it suggests to be "protected".  Is there a reason for making them "internal" ?